.github/workflows/test-zombienet.yml

```
@@ -0,0 +1,49 @@
+name: Test Zombienet
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-zombienet:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

There is only a single element in the matrix, so we can simply set up the value

```suggestion
    runs-on: ubuntu-latest
```